molSimplify/job_manager/resub.py

```diff
@@ -289,14 +289,14 @@ def resub(directory='in place'):
 def resub_psi4(psi4_config):
     basedir = os.getcwd()
     if "trigger" in psi4_config:
-        write_jobscript(psi4_config)
-        if not "cluster" in psi4_config or psi4_config["cluster"] == "mustang":
+        manager_io.write_jobscript(psi4_config)
```

Comment:
@chenruduan also here.

Reply:
Here should be okay since `write_jobscript` is a function in `manager_io`.

Reply:
Okay, I was wrong. The `write_jobscript` in `manager_io` is not the same function as that in `psi4_utils.run` (we have two functions sharing the same name...). The safest way is to add a local import here since this `resub_psi4` is psi4 workflow only.

Reply:
I don't know why exactly you made the changes at around line 14. But if you don't like `run_bash` being imported globally, I think we can add it here as local import as well.

Reply:
The reason the change was necessary in the first place is because `write_jobscript` and `run_bash` are currently not imported (that why the linter refers to them as undefined). I really do not have a preference for how this is resolved. The main argument for using `psiutils.run.run_bash` and `manager_io.write_jobscript` instead of the plain import is exactly the fact that there are 2 versions of both of these functions within molSimplify.

Reply:
The easiest fix would be to uncomment line 14. But there must be a reason someone commented that line in the first place, right?

Reply:
Manged to track down the commit: 78a55f77b90602df02b057147563e0386fed2384 
Lets just assume this was unintentional and revert to the previous state!

Reply:
It looks like unintentional to me. @IsabeltheCat Do you have an idea about why you commented that line? But Haojun once experienced some difficulties of getting both molSimplify and Psi4 to work together. Probably we can uncomment that line for now. I will look into the condo_envs.yaml file and see how to fix it.